omnigibson/robots/manipulation_robot.py

```diff
@@ -916,6 +916,7 @@ def _freeze_gripper(self, arm="default"):
         for joint_name, j_val in self._ag_freeze_joint_pos[arm].items():
             joint = self._joints[joint_name]
             joint.set_pos(pos=j_val)
+            joint.set_vel(vel=0.0)
```

Comment:
why is this needed?

Reply:
This used to be here but was removed recently by Wensi attempting to fix this issue. This solves it so we can put it back. It stops the gripper from moving out.